sdk/core/core-client/src/serializer.ts

```diff
@@ -430,7 +430,8 @@ function serializeBasicTypes(typeName: string, objectName: string, value: any):
       const objectType = typeof value;
       if (
         objectType !== "string" &&
-        typeof value.pipe !== "function" &&
+        typeof value.pipe !== "function" && // NodeJS.ReadableStream
+        typeof value.tee !== "function" &&  // browser ReadableStream
```

Comment:
should we update the below error message to take out the 'nodeJS' prefix since it supports either browser or node streams now?

Reply:
Yes good point!